### PR TITLE
Make dyload strictly use the same ABI in header

### PR DESCRIPTION
**Pull request by reyoung**

### CODE REVIEW

File: paddle/fluid/platform/dynload/cublas.h [hunk 1]
```diff
@@ -71,8 +73,8 @@ extern void *cublas_dso_handle;
   __macro(cublasDgemm_v2);                \
   __macro(cublasHgemm);                   \
   __macro(cublasSgemmEx);                 \
-  __macro(cublasSgeam_v2);                \
-  __macro(cublasDgeam_v2);                \
+  __macro(cublasSgeam);                   \
+  __macro(cublasDgeam);                   \
```
Comment (panyx0718) [inline paddle/fluid/platform/dynload/cublas.h]: why is this changed?

Comment (reyoung) [reply paddle/fluid/platform/dynload/cublas.h]: Because there is no API named `cublasDgeam_v2` in cublas. It is a bug before.
